.github/workflows/packaging.yml

```
@@ -167,8 +168,9 @@ jobs:
           echo "schedule: ${{ github.event.schedule }}" >> $GITHUB_STEP_SUMMARY
           echo "ref: ${{ github.ref }}" >> $GITHUB_STEP_SUMMARY
 
-      - uses: actions/download-artifact@v3
+      - uses: actions/download-artifact@v4
         with:
+          name: go-ubuntu-latest
```

Not sure how this worked out with the clobbering? Lucky timing maybe?

---

(`merge_multiple: true` was the silent default)